src/main/java/randoop/main/GenTests.java

```diff
@@ -537,6 +612,42 @@ public boolean handle(String[] args) {
     return true;
   }
 
+  /**
+   * Counts the number of sequences an operation occurs in. A sequence contributes 1 to the tally of
+   * operation Op if Op appears at least once in the sequence. Further occurrences in the same
+   * sequence (past the initial occurrence) do not contribute to the tally.
+   *
+   * @param sequences sequences to process for operations
+   * @return the output count map to increment for each operation. For each operation, the map maps
+   *     to the number of sequences in which the operation occurs at least once.
+   */
+  private Map<TypedOperation, Integer> tallyOperationsInSequences(
+      List<ExecutableSequence> sequences) {
+    Map<TypedOperation, Integer> tallyMap = new HashMap<>();
+
+    for (ExecutableSequence es : sequences) {
+      // Ensure that each operation is only counted once per sequence.
```

Comment:
I suggest abstracting out the first half of the body of this loop into a method named something like `operationsInSequence`.  That will make the rest of the logic clearer.